keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -33,6 +35,35 @@ open class SqlResourceRepository(
   private val objectMapper: ObjectMapper
 ) : ResourceRepository {
 
+  override fun deleteByApplication(application: String) {
```

Comment:
@robfletcher Please lmk if there's a better way of writing this - I wanted to first check if there are any applications that exist with `application` as the name, and throw an exception if not prior to actually deleting those records.

Reply:
Nevermind :) just figured it out.